components/CompletionScoreCard.tsx

```diff
@@ -0,0 +1,260 @@
+import { ArrowUpRight, CircleFill } from 'react-bootstrap-icons';
+import MOCK_MEASURES from '@/mocks/measures.json';
+
+interface CompletionScoreCardProps {}
+
+interface MockData {
+  title: string;
+  linkText: string;
+  linkHref: string;
+  scoreData: ScoreData[];
+}
+
+interface ScoreData {
+  priority: 'High' | 'Moderate' | 'Low';
+  total: number;
+  completed: number;
+}
+
+interface CalculatedScore {
+  priority: string;
+  score: string;
+}
+
+const mockData = {
+  title: 'Zero COventry',
+  linkText: 'netzerocities.kausal.tech/coventry-netzero',
+  linkHref: '#',
+};
+
+const processDataCollection = () => {
+  const result = [
+    { priority: 'High', total: 0, completed: 0 } as ScoreData,
+    { priority: 'Moderate', total: 0, completed: 0 } as ScoreData,
+    { priority: 'Low', total: 0, completed: 0 } as ScoreData,
+  ];
+
+  const processItems = (items: any[]) => {
+    items.forEach((item) => {
+      if (item.priority === 'HIGH') {
+        result[0].total += 1;
+        if (item.value) result[0].completed += 1;
+      } else if (item.priority === 'MODERATE') {
+        result[1].total += 1;
+        if (item.value) result[1].completed += 1;
+      } else if (item.priority === 'LOW') {
+        result[2].total += 1;
+        if (item.value) result[2].completed += 1;
+      }
+      if (item.items) {
+        processItems(item.items);
+      }
+    });
+  };
+
+  MOCK_MEASURES['dataCollection'].items.forEach((section) => {
+    processItems(section.items);
+  });
+
+  return result;
+};
+
+const calculateScores = (scoreData: ScoreData[]) => {
+  return scoreData.map((data) => ({
+    priority: data.priority,
+    score: `${data.completed}/${data.total}`,
+  }));
+};
+
+const calculatePercentage = (scoreData: ScoreData[]) => {
+  const coeff: { [key: string]: number } = {
+    High: 0.6,
+    Moderate: 0.3,
+    Low: 0.1,
+  };
+  const totalPercentage = scoreData.reduce((acc, data) => {
+    const completionPercentage =
+      data.total > 0 ? data.completed / data.total : 0;
+    return acc + completionPercentage * (coeff[data.priority] || 0);
+  }, 0);
+
+  return totalPercentage * 100;
+};
+
+export const CompletionScoreCard: React.FC<CompletionScoreCardProps> = () => {
+  const [data, setData] = useState<MockData | null>(null);
+  const [scores, setScores] = useState<CalculatedScore[]>([]);
+  const [completionPercentage, setCompletionPercentage] = useState<number>(0);
+
+  const theme = useTheme();
+
+  useEffect(() => {
+    setTimeout(() => {
+      const scoreData = processDataCollection();
+      const finalMockData: MockData = {
+        ...mockData,
+        scoreData: scoreData,
+      };
+
+      setData(finalMockData);
+      const calculatedScores = calculateScores(scoreData);
+      setScores(calculatedScores);
+
+      const percentage = calculatePercentage(scoreData);
+      setCompletionPercentage(percentage);
+    }, 1000);
+  }, []);
+
+  if (!data) {
+    return <Typography>Loading...</Typography>;
+  }
```

Comment:
No need to add this now, but just FYI, MUI has handy loading components for cases like this

https://mui.com/material-ui/react-skeleton/